<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="49" started-at="2018-12-29T00:59:21Z" finished-at="2018-12-29T00:59:21Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="49" started-at="2018-12-29T00:59:21Z" finished-at="2018-12-29T00:59:21Z">
      <class name="day8.DataProviderTesting">
        <test-method status="FAIL" signature="printAllData(java.lang.String, java.lang.String)[pri:0, instance:day8.DataProviderTesting@7ac7a4e4]" name="printAllData" duration-ms="0" started-at="2018-12-29T00:59:21Z" data-provider=" getData" finished-at="2018-12-29T00:59:21Z">
          <exception class="org.testng.TestNGException">
            <message>
              <![CDATA[
Method public void day8.DataProviderTesting.printAllData(java.lang.String,java.lang.String) requires a @DataProvider named :  getData]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.testng.TestNGException: 
Method public void day8.DataProviderTesting.printAllData(java.lang.String,java.lang.String) requires a @DataProvider named :  getData
at org.testng.internal.Parameters.findDataProvider(Parameters.java:513)
at org.testng.internal.Parameters.handleParameters(Parameters.java:691)
at org.testng.internal.ParameterHandler.handleParameters(ParameterHandler.java:49)
at org.testng.internal.ParameterHandler.createParameters(ParameterHandler.java:37)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:924)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- org.testng.TestNGException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- printAllData -->
      </class> <!-- day8.DataProviderTesting -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
